distributed/protocol/numpy.py

```diff
@@ -47,15 +47,17 @@ def serialize_numpy_ndarray(x):
               'strides': x.strides,
               'shape': x.shape}
 
+    data = x.view('u1').data
```

Comment:
Yeah, I think that these would all be tricky situations if the itemsizes were relatively prime or if you actually cared about indexing afterwards. I think we're safe turning it into a view of singleton bytes though.